docker/install/ubuntu_install_caffe.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+set -u
+set -o pipefail
+
+apt-get update --fix-missing
+
+# avoid manually selecting the time zone
+export DEBIAN_FRONTEND=noninteractive
+apt-get install -y tzdata
```

Comment:
Can you comment about the time zone setup and whether it is necessary?

Reply:
When installing pre-compiled caffe, it first installs the tzdata dependency, and then you need select the time zone manually. In other words, it is neccessary to install precompiled caffe.

Reply:
please add a more detailed comment about this case then. thank you

Reply:
Ok，i will add more comments on it later. By the way, do you know why this pr cant not pass the ci tests? The following information shows that the it pass the buiding windows-latest but fail to build macOS-latest. When i click the "Details", it just display the "This check failed", and no more other info.

Reply:
please re-trigger it again. Thanks.